tests/testthat/_snaps/unix/helper-functions.md

```diff
@@ -0,0 +1,25 @@
+# `ui_throw()` called by `rust_function()` captures Rust compilation errors
+
+    Rust code could not be compiled successfully. Aborting.
+    x error[E0412]: cannot find type `i33` in this scope
+     --> src/lib.rs:3:19
```

Comment:
Are you sure you want to snapshot these error messages? Any time something minor changes in rustc's output this will break.

Reply:
I understand the danger, yet I trust Rust developer team and believe they will not change the output of `cargo`/`rustc`. Even if they will (e.g., if something like Rust 2.0 is released), I see no problem in updating snapshots. After all, every time `{rlang}`, `{cli}`, `{knitr}`, or any other library that produces formatted output has something changed under the hood, we are forced to update the snapshots (as soon as we notice). This also draws our attention to potential changes that we overlook.

Reply:
Ok, sounds good.